assets/css/less/admin-settings.less

```diff
@@ -258,6 +258,12 @@
     }
 }
 
+/* Using the ID selector is necessary because there is no class-based identifier for this settings */
+.wpbdp-no-renewal #wpbdp-settings-admin-notifications .wpbdp-settings-multicheck-option-no-2,
+.wpbdp-no-renewal #wpbdp-settings-admin-notifications .wpbdp-settings-multicheck-option-no-3,
+.wpbdp-no-renewal #wpbdp-settings-user-notifications .wpbdp-settings-multicheck-option-no-3 {
```

Comment:
Let's take advantage of the Less format here to avoid repeating the .wpbdp-no-renewal in each line.